packages/babel-preset-env/src/normalize-options.js

```diff
@@ -128,13 +128,16 @@ export const validateIgnoreBrowserslistConfig = (
   );
 
 export const validateModulesOption = (
-  modulesOpt: ModuleOption = ModulesOption.commonjs,
+  modulesOpt: ModuleOption = ModulesOption.auto,
 ) => {
   invariant(
     ModulesOption[modulesOpt] ||
       ModulesOption[modulesOpt] === ModulesOption.false,
-    `Invalid Option: The 'modules' option must be either 'false' to indicate no modules, or a
-    module type which can be be one of: 'commonjs' (default), 'amd', 'umd', 'systemjs'.`,
+    `Invalid Option: The 'modules' option must be one of \n` +
+      ` - 'false' to indicate no module processing\n` +
+      ` - a specific module type: 'commonjs', 'amd', 'umd', 'systemjs'` +
+      ` - 'auto' (default) which will automatically select 'false' if the current\n` +
+      `   process is known to support ES module syntax, or "commonjs" otherwise\n`,
```

Comment:
Hi, I have a ❓ about "if the current process is known to support ES module syntax". I'm not sure what that means. Can someone please clarify? Thank you!

Reply:
For example, if you are calling Babel using `babel-loader`, `modules` will be set to false because webpack supports ES modules

Reply:
Thanks @nicolo-ribaudo, I'll link to your reply from https://github.com/babel/babel-loader/issues/521 😃